packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/cells/value-cell.tsx

```diff
@@ -19,8 +25,26 @@ export const renderValueCell: CustomRenderer<ValueCell> = {
     ctx.font = theme.baseFontStyle;
 
     const yCenter = getYCenter(args);
+    const left = rect.x + getCellHorizontalPadding();
+
+    if (isValueEmpty(value)) {
+      ctx.fillStyle = "#91A5BA";
+      ctx.font = "italic 14px Inter";
+      return ctx.fillText("No value", left, yCenter);
+    }
+
+    if (typeof value === "boolean") {
+      return drawTextWithIcon({
+        args,
+        text: capitalize(String(value)),
```

Comment:
This again feels too clever to me – can we just do 

```suggestion
        text: value ? 'True' : 'False',
```